geom/src/polygon.rs

```diff
@@ -292,7 +292,13 @@ impl Polygon {
 
     // TODO Result won't be a nice Ring
     pub fn intersection(&self, other: &Polygon) -> Vec<Polygon> {
-        from_multi(to_geo(self.points()).intersection(&to_geo(other.points())))
+        let geo_polygon = to_geo(self.points());
```

Comment:
I'm not sure about this, I played around with A/B Street a bit and it doesn't seem to have broken anything in the tutorial but I can't be certain. What is this function really meant to do?

Reply:
Looks like this is used in just one place: `convert_osm/src/clip.rs`, to take area polygons (like water and grass) and clip them to the map's bounding polygon. This is important enough to keep.

I haven't tried in a while; what happens if you include the latest version of geon-booleanop with these changes? Do we need to go update that crate to depend on the latest geo-types or something?